### PR TITLE
DAT-98-update-download-endpoint

### PR DESCRIPTION
**Pull request by jhoffman-ms**
Updated download invoice endpoint (`/accounts/{origin}/{clientId:int:min(1)}/invoice/{filename}`) to returns a InvoiceFile object:

Now Json result:

```json
{
    "contentType": "application/pdf",
    "content": "BERi0xLjQKJbS1trcKJQoxIDAgb2JqCjw8Ci9EZXN0cyAyIDAgUgovUGFnZU1vZGUgL1VzZU5vbmUKL1BhZ2VzIDMgMC BSCi9UeXBlIC9DYXRhbG9nCi9WaWV3ZXJQcmVmZXJlbmNlcyA0IDAgUgovUGFnZUxheW91dCAvT25lQ29sdW1uCj4+CgplbmRvYmoKMyAwIG9iago8PAovQ291bnQgMQovS2lkcyBbNSAwIFJdCi9UeXBlIC9QYWdlcwo+PgoKZW5kb2JqCjUgMCBvYmoKPDwKL0NvbnRlbnRzIFs2IDAgUl0KL1BhcmVudCAzIDAgUgovVHlwZSAvUGFnZQovTWV..."
}
```

### CODE REVIEW

**Comment by andresmoschini** [comment]
Oh, it should be a misunderstanding.

If I understand it well, returning a raw file is better than wrap it into a JSON. Why did you apply this change?

File: Billing.API/Controllers/InvoiceController.cs [hunk 1]
```diff
@@ -55,7 +56,7 @@ public async Task<IActionResult> GetInvoices([FromRoute] string origin, [FromRou
             if (response == null)
                 return NotFound();
 
-            return File(response, "application/pdf", filename);
+            return Ok(response);
```
Comment (rockambole86) [inline Billing.API/Controllers/InvoiceController.cs:59]: Why changing a File to an Ok response? Is it for something special?

Comment (andresmoschini) [reply Billing.API/Controllers/InvoiceController.cs:59]: I agree, I really prefer a raw response with the file content than a JSON, but maybe we are missing something, let's discuss it after the standup.

**Comment by jhoffman-ms** [comment]
was my mistake. Sorry I closed the PR

**Pull request closed without merging**